odl_video/settings.py

```diff
@@ -531,3 +531,5 @@ def get_all_config_keys():
 EDX_ACCESS_TOKEN = get_string("EDX_ACCESS_TOKEN", None)
```

Comment:
Should this also be removed since it's no longer being used? Think that gets written to the db at some point too so might need a migration